kairon/shared/pos/processor.py

```
@@ -0,0 +1,646 @@
+        list_payload = {
+            "jsonrpc": "2.0",
+            "method": "call",
+            "params": {
+                "service": "db",
+                "method": "list",
+                "args": []
+            },
+            "id": 1
+        }
+
+        dbs = requests.post(url, json=list_payload).json().get("result", [])
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)